packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsForm/PayableDetailsForm.tsx

```diff
@@ -141,7 +141,7 @@ const getValidationSchema = (i18n: I18n) =>
  * @example MUI theming
  * const theme = createTheme({
  *   components: {
- *     MonitePayableDetailsForm: {
+ *     MonitePayableDetailsInfo: {
```

Comment:
what is the reason of this renaming? The component is still called PayableDetailsForm

Reply:
Moved to private discussion